app/grandchallenge/api/swagger.py

```diff
@@ -0,0 +1,49 @@
+from typing import Dict, List, Tuple, Union
+
+from drf_yasg.inspectors import FieldInspector
+from drf_yasg.openapi import Schema
+
+
+def add_manual_fields(self, serializer_or_field, schema):
+    meta = getattr(serializer_or_field, "Meta", None)
+    swagger_schema_fields = getattr(meta, "swagger_schema_fields", {})
+    if swagger_schema_fields:
+        update_swagger_schema_fields(schema, swagger_schema_fields)
+
+
+def update_swagger_schema_fields(schema, swagger_schema_fields):

```

Comment:
These should probably be protected functions, prepend a `_` to both.

Reply:
OK.